docs/dashboards/introduction.md

```diff
@@ -94,6 +94,28 @@ Link cards are specialized cards that let you search and link to other items in
 
 To add a link card to a dashboard, click the **pencil** icon to enter dashboard editing mode, then click on the **link** icon. Click on the input field in the link card to search your Metabase for an item to link to, or paste an external link.
 
+## Iframe cards
+
+[Iframes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) allow you to nest one HTML page in another HTML page. With iframe cards in Metabase, you can embed a webpage in your dashboard: a video, online spreadsheet, whatever.
+
+To add an iframe card to a dashboard, click the **pencil** icon to enter dashboard editing mode, then click on the **link** icon and select **Iframe**. Then paste the iframe URL. Here's an iframe for our video on how to create a dashboard:
+
+```html
+<iframe width="560" height="315" src="https://www.youtube.com/embed/W-i9E5_Wjmw?si=qWeMGxEymVOIHiXo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+```
+
+And behold, a dashboard with an embedded video:
+
+![Dashboard with iframe of video](./images/dashboard-with-iframe.png)
+
+Wait, does this mean that you can you [embed Metabase](../embedding/introduction.md) _in_ Metabase?
```

Comment:
I wouldn't seed the idea of embedding Metabase in Metabase. It's fun, but not particularly useful, and I'm worried it's bound to open up some weird interactions, so I don't want to nudge people towards it. I think more useful would be some practical idea starters:
- Spreadsheets, Surveys for data entry or workflows
- Videos for contextualizing and storytelling on dashboards
- Charts from other tools